modules/ROOT/pages/storage.adoc

```diff
@@ -129,9 +163,51 @@ storage:
       wipe_filesystem: true
       label: root
 ----
-
 NOTE: You don't need the `path` or `with_mount_unit` keys; FCOS knows that the root partition is special and will figure out how to find it and mount it.
 
+.Defining a file system on a RAID storage device
```

Comment:
Needs a sentence or two explaining what the example does.  Also, this example should get its own section, rather than being grouped under root filesystem moving.

Reply:
Done.